### PR TITLE
Moved `Element` construction to a static `Optional` method

### PR DESCRIPTION
**Pull request by amuradyan**

### CODE REVIEW

**Review by Karlengevorgyan91 — APPROVED** [review]

File: src/test/java/shwxr7/model/ElementTests.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  @DisplayName("The name of the element cannot be empty")
+  public void the_name_of_the_element_cannot_be_empty() {
+    var badElement = Element.of("");
+
+    assertTrue(badElement.isEmpty());
+  }
+
+  @Test
+  @DisplayName("One shuold be able to create an element with valid name and id")
+  public void one_should_be_able_to_create_an_element_with_valid_name_and_id() {
```
Comment (VazgenLevonyan) [inline src/test/java/shwxr7/model/ElementTests.java:29]: I think, the method name should be "one_should_be_able_to_create_an_element_with_valid_name" as we have decided, that for creating an object of Element, we should provide only  the "name" of that object.

File: src/test/java/shwxr7/model/DiscConfigTests.java [hunk 2]
```diff
@@ -13,11 +13,11 @@
 
 public class DiscConfigTests {
 
-  private final Element _10 = new Element("10", 10);
-  private final Element J = new Element("J", 11);
-  private final Element Q = new Element("Q", 12);
-  private final Element K = new Element("K", 13);
-  private final Element A = new Element("A", 0);
+  private final Element _10 = Element.of("1").get();
```
Comment (VazgenLevonyan) [inline src/test/java/shwxr7/model/DiscConfigTests.java:16]: If I am not mistaken, the name of _10 object, instead of "1"  should be "10".

**Review by VazgenLevonyan — CHANGES_REQUESTED** [review]

File: src/test/java/shwxr7/model/DiscConfigTests.java (hunk 2 above)
Comment (amuradyan) [reply src/test/java/shwxr7/model/DiscConfigTests.java:16]: Java does not allow for numbers as valid names.

File: src/test/java/shwxr7/model/ElementTests.java (hunk 1 above)
Comment (amuradyan) [reply src/test/java/shwxr7/model/ElementTests.java:29]: This is true. Unfortuna I have merged the PR already. I'll fix this in a new PR.

Thanks a lot.

File: src/test/java/shwxr7/model/DiscConfigTests.java (hunk 2 above)
Comment (amuradyan) [reply src/test/java/shwxr7/model/DiscConfigTests.java:16]: I misread the comment, sorry. I'll Fix this in an upcoming PR as well.

ty

**Pull request merged**